lookup_plugin/helper.py

```diff
@@ -0,0 +1,195 @@
+import json
+import os  
+import shutil
+import random
+from ansible.plugins.lookup import LookupBase
+
+DBI_DIR = "dbi-dbo"
+GEN_NUM = 1
+Marker_vdev = 0
+Marker_chunk = 1
+Marker_type = 4
```

Comment:
we can rename constants using all caps
MARKER_VDEV  # marker vdev 
MARKER_CHUNK  # marker chunk
MARKER_TYPE     # marker type

or MKR_VDEV ... etc